### PR TITLE
adduser: add German translation

### PR DESCRIPTION
**Pull request by lemeleti**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/linux/adduser.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# adduser
+
+> Tool um Benutzer hinzuzufügen.
+> Mehr Informationen: <https://manpages.debian.org/latest/adduser/adduser.html>.
+
+- Erzeugen eines neuen Benutzers mit einem Standard-Home-Verzeichnis und Aufforderung an den Benutzer, ein Passwort festzulegen:
+
+`adduser {{username}}`
```
Comment (navarroaxel) [inline pages.de/linux/adduser.md:8]: Please translate placeholders to help non-English speakers
```suggestion
`adduser {{benutzername}}`
```

File: pages.de/linux/adduser.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Erzeugen eines neuen Benutzers ohne Home-Verzeichnis:
+
+`adduser --no-create-home {{username}}`
```
Comment (navarroaxel) [inline pages.de/linux/adduser.md:12]:
```suggestion
`adduser --no-create-home {{benutzername}}`
```

File: pages.de/linux/adduser.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`adduser --home {{path/to/home}} {{username}}`
+
+- Erzeugen eines neuen Benutzers, bei dem die angegebene Shell als Anmeldeshell eingestellt ist:
+
+`adduser --shell {{path/to/shell}} {{username}}`
+
+- Erzeugen eines neuen Benutzers, der zur angegebenen Gruppe gehört:
+
+`adduser --ingroup {{group}} {{username}}`
```
Comment (navarroaxel) [inline pages.de/linux/adduser.md:24]:
```suggestion
`adduser --ingroup {{gruppe}} {{benutzername}}`
```

File: pages.de/linux/adduser.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Hinzufügen eines vorhandenen Benutzers zur angegebenen Gruppe:
+
+`adduser {{username}} {{group}}`
```
Comment (navarroaxel) [inline pages.de/linux/adduser.md:28]:
```suggestion
`adduser {{benutzername}} {{gruppe}}`
```

File: pages.de/linux/adduser.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`adduser --no-create-home {{username}}`
+
+- Erzeugen eines neuen Benutzers mit einem Home-Verzeichnis unter dem angegebenen Pfad:
+
+`adduser --home {{path/to/home}} {{username}}`
+
+- Erzeugen eines neuen Benutzers, bei dem die angegebene Shell als Anmeldeshell eingestellt ist:
+
+`adduser --shell {{path/to/shell}} {{username}}`
```
Comment (navarroaxel) [inline pages.de/linux/adduser.md:20]:
```suggestion
`adduser --shell {{pfad/zu/shell}} {{benutzername}}`
```

File: pages.de/linux/adduser.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`adduser {{username}}`
+
+- Erzeugen eines neuen Benutzers ohne Home-Verzeichnis:
+
+`adduser --no-create-home {{username}}`
+
+- Erzeugen eines neuen Benutzers mit einem Home-Verzeichnis unter dem angegebenen Pfad:
+
+`adduser --home {{path/to/home}} {{username}}`
```
Comment (navarroaxel) [inline pages.de/linux/adduser.md:16]:
```suggestion
`adduser --home {{pfad/zu/home}} {{benutzername}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages.de/linux/adduser.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Erzeugen eines neuen Benutzers, bei dem die angegebene Shell als Anmeldeshell eingestellt ist:
+
+`adduser --shell {{pfad/zu/shell}} {{benutzername}}`
+
+- Erzeugen eines neuen Benutzers, der zur angegebenen Gruppe gehört:
+
+`adduser --ingroup {{gruppe}} {{benutzername}}`
+
+- Hinzufügen eines vorhandenen Benutzers zur angegebenen Gruppe:
```
Comment (marchersimon) [inline pages.de/linux/adduser.md:26]:
```suggestion
- Füge einen vorhandenen Benutzer zur angegebenen Gruppe hinzu:
```

File: pages.de/linux/adduser.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Erstelle einen neuen Benutzer mit einem Home-Verzeichnis unter dem angegebenen Pfad:
+
+`adduser --home {{pfad/zu/home}} {{benutzername}}`
+
+- Erstelle einen neuen Benutzer, bei dem die angegebene Shell als Anmeldeshell eingestellt ist:
+
+`adduser --shell {{pfad/zu/shell}} {{benutzername}}`
+
+- Erzeuge einen neuen Benutzer und füge ihn zur angegebenen Gruppe hinzu:
```
Comment (marchersimon) [inline pages.de/linux/adduser.md:22]:
```suggestion
- Erstelle einen neuen Benutzer und füge ihn zur angegebenen Gruppe hinzu:
```

**Comment by lemeleti** [comment]
Used wrong commit message format. That's why theres a force-push.

**Comment by marchersimon** [comment]
@lemeleti the commit message doesn't matter. Whoever merges the PR can set the commit message

**Review by marchersimon — APPROVED** [review]
Thanks

**Pull request merged**